MediaBrowser.Common/Net/NetworkExtensions.cs

```diff
@@ -27,9 +27,10 @@ public static void AddItem(this Collection<IPObject> source, IPAddress ip)
         /// </summary>
         /// <param name="source">The <see cref="Collection{IPObject}"/>.</param>
         /// <param name="item">Item to add.</param>
-        public static void AddItem(this Collection<IPObject> source, IPObject item)
+        /// <param name="unique"><c>True</c> if subnets that overlap should be merged (default).</param>
+        public static void AddItem(this Collection<IPObject> source, IPObject item, bool unique = true)
         {
-            if (!source.ContainsAddress(item))
+            if (!source.ContainsAddress(item) || !unique)
```

Comment:
Does this check that the IP is not the exact same?

Reply:
changed the var name and description.

True means 10.1.1.1/24 and 10.1.1.2/24 would both be added as they are addresses.
False means 10.1.1.1/24 and 10.1.1.2/24 would not add the 2nd as the first will be treated as the network 10.1.1.0/24 of which 10.1.1.2/24 is part.